### ISSUE TITLE
Clay Card 3.x: stickerProps doesn't support all the properties Clay Card 2.x had

### ISSUE THREAD

**Issue by julien** [liferay/clay#3778]
In 3.x, the Card's `stickerProps` property only supports [`content` and `displayType`](https://github.com/liferay/clay/blob/6b00a1e60deaa3c415dbed5227c3ca6017ac14ea/packages/clay-card/src/CardWithInfo.tsx#L90-L93), I already opened #3773 concerning images, but we're also missing [these](https://github.com/liferay/clay/blob/3bd63bb5b84a60b2cd1da4746bb05f3874f1d415/packages/clay-card/src/ClayCardBase.soy#L32-L38) from 2.x:

- `stickerCssClass`
- `stickerIcon` (as far as I can see this [always has a value of `document-text`](https://github.com/liferay/clay/blob/2c554a9338c4d6b1c22d6b73873433e1a0c029f5/packages/clay-card/src/CardWithInfo.tsx#L158))
- `stickerImageAlt`
- `stickerImageSrc`
- `stickerLabel`
- `stickerShape`

My use case is that the `<clay:vertical-card />` JSP tag [supports these attributes](https://github.com/liferay/liferay-portal/blob/8dd8349f3a2ff73ae02eb15adef344286e575fdb/modules/apps/frontend-taglib/frontend-taglib-clay/src/main/resources/META-INF/liferay-clay.tld#L3739-L3780) and we'd like to avoid breaking changes as much as possible.

[EDIT]: More information about the properties priority can be found [here](https://github.com/liferay-frontend/liferay-portal/pull/448#discussion_r512153368)

**Comment by julien** [comment]
In the the [React component](https://github.com/liferay-frontend/liferay-portal/blob/f712cda66d1a301c15ba16002146783ca6b22638/modules/apps/frontend-taglib/frontend-taglib-clay/src/main/resources/META-INF/resources/cards/VerticalCard.js#L55-L63) that the [java tag](https://github.com/liferay-frontend/liferay-portal/blob/c49455f79c77d58ea780b8c0c4bfe28f620b120a/modules/apps/frontend-taglib/frontend-taglib-clay/src/main/java/com/liferay/frontend/taglib/clay/servlet/taglib/VerticalCardTag.java) uses, we will use the following:

```js
{
  className: stickerCssClass,
  content: stickerLabel,
  displayType: stickerStyle,
  imageAlt: stickerImageAlt,
  imageSrc: stickerImageSrc,
  shape: stickerShape,
   symbol: stickerIcon,
}
``` 

If you want to change those names once the change is done in the component no problem as far as we (@carloslancha @markocikos and I) are concerned, but please *don't forget* to update the java tag and it's component.

thanks

**Comment by carloslancha** [comment]
Closing this in favor of https://github.com/liferay/clay/issues/3773
